source/config/configFlags.py

```diff
@@ -0,0 +1,37 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Cyrille Bougot
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+"""Flags used for the configuration.
+"""
```

Comment:
[Python's conventions for docstrings](https://peps.python.org/pep-0257/) recommends to place closing triple quotes on the same line as the docstring if it is only a single line long.

Reply:
Done in 450cfec36e9fca0c58713c7ce35ac32565296beb.